committeeoversightapp/templates/partials/top_menu.html

```diff
@@ -0,0 +1,45 @@
+{% load static wagtailcore_tags cms_tags %}
+
+<nav class="navbar navbar-dark fixed-top footer navbar-expand-md">
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></i></span>
+  </button>
+  <a class="navbar-brand" href="/">
+    <img src="{% static 'images/lugarcenter.svg' %}" height="40" alt="The Lugar Center"/>
+  </a>
+  <div class="collapse navbar-collapse" id="navbarNav">
+    <ul class="navbar-nav ml-auto">
+      <li class="nav-item">
+        <a href="/">Home</a>
+      </li>
+      {% for menuitem in menuitems %}
+        <li class="nav-item {% if menuitem.show_dropdown %}dropdown{% endif %}{% if calling_page.url == menuitem.url %} active{% endif %}">
+            {% if menuitem.show_dropdown %}
+              <a data-toggle="dropdown" class="dropdown-toggle" href="#">{{ menuitem.title }} <b class="caret"></b></a>
+              {% top_menu_children parent=menuitem %}
+            {% else %}
+              <a href="{% pageurl menuitem %}">{{ menuitem.title }}</a>
+            {% endif %}
+        </li>
+      {% endfor %}
+      {% if request.user.is_superuser %}
+      <li class="nav-item">
+        <a href="{% url 'create-event'%}">Add Hearing</a>
+      </li>
+      <li class="nav-item">
+        <a href="{% url 'list-event'%}">View Hearings</a>
+      </li>
+      {% endif %}
+      {% if request.user.is_authenticated %}
+      <li class="nav-item">
+        <span class="mr-4">Logged in as <b>{{ request.user.get_username }}</b></span>
```

Comment:
What do you think about changing the `<span` here to `<small>` and putting this in italics? 

```suggestion
        <small class="mr-4 font-italic">Logged in as <b>{{ request.user.get_username }}</b></small>
        
```

It would look like this: 

<img width="810" alt="Screen Shot 2019-09-23 at 3 45 53 PM" src="https://user-images.githubusercontent.com/1094243/65461302-3f5b2f80-de19-11e9-8b19-b55c5fd1a8f8.png">